extensions/wikia/CommunityPage/helpers/WikiTopic.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class WikiTopic {
+	public static function prepareWikiTopic( $siteName ) {
+		return trim( preg_replace( '/Wiki$/i', '', $siteName ) );
```

Comment:
The spreadsheet strips "wiki" or "wikia" at the end, and we should probably do the same here. And per Lukasz, the check on whether to return `$wgWikiTopic` should probably go in a getter in this class.